docs/detections/rules-ui-monitor.asciidoc

```diff
@@ -112,65 +112,43 @@ image::images/timestamp-override.png[]
-events, use the Security:Windows {ml} jobs. If you want alerts for anomalies in
-these jobs, make clones of the existing {ml} rules and update them to use these
-jobs.
-
-If you are cloning prebuilt {ml} rules to generate alerts for the new {ml} jobs,
-the following rules are affected:
-
-* <<unusual-linux-network-port-activity>>: Use
-`v2_linux_anomalous_network_port_activity_ecs` instead of
-`linux_anomalous_network_port_activity_ecs`.
-* <<anomalous-process-for-a-linux-population>>: Use
-`v2_linux_anomalous_process_all_hosts_ecs` instead of
-`linux_anomalous_process_all_hosts_ecs`.
-* <<unusual-linux-username>>: Use `v2_linux_anomalous_user_name_ecs` instead of
-`linux_anomalous_user_name_ecs`.
-* <<unusual-linux-process-calling-the-metadata-service>>: Use
-`v2_linux_rare_metadata_process` instead of `linux_rare_metadata_process`.
-* <<unusual-linux-user-calling-the-metadata-service>>: Use
-`v2_linux_rare_metadata_user` instead of `linux_rare_metadata_user`.
-* <<unusual-process-for-a-linux-host>>: Use `v2_rare_process_by_host_linux_ecs`
-instead of `rare_process_by_host_linux_ecs`.
-* <<unusual-process-for-a-windows-host>>: Use
-`v2_rare_process_by_host_windows_ecs` instead of
-`rare_process_by_host_windows_ecs`.
-* <<unusual-windows-network-activity>>: Use
-`v2_windows_anomalous_network_activity_ecs` instead of
-`windows_anomalous_network_activity_ecs`.
-* <<unusual-windows-path-activity>>: Use `v2_windows_anomalous_path_activity_ecs`
-instead of `windows_anomalous_path_activity_ecs`.
-* <<anomalous-windows-process-creation>>: Use
-`v2_windows_anomalous_process_creation` instead of
-`windows_anomalous_process_creation`.
-* <<anomalous-process-for-a-windows-population>>: Use
-`v2_windows_anomalous_process_all_hosts_ecs` instead of
-`windows_anomalous_process_all_hosts_ecs`.
-* <<unusual-windows-username>>: Use `v2_windows_anomalous_user_name_ecs` instead
-of `windows_anomalous_user_name_ecs`.
-* <<unusual-windows-process-calling-the-metadata-service>>: Use
-`v2_windows_rare_metadata_process` instead of `windows_rare_metadata_process`.
-* <<unusual-windows-user-calling-the-metadata-service>>: Use
-`v2_windows_rare_metadata_user` instead of `windows_rare_metadata_user`.
+{ml-cap} detection rules use {ml} jobs that have dependencies on data fields populated by the {beats} and {agent} integrations. In version 8.3, new {ml} jobs (prefixed with `V3`) were released to operate on the ECS fields available at that time. If you're using multiple versions of {beats} or {agent} (from both before and after 8.3), you may need to duplicate or create new {ml} rules that specify the new ML `V3` jobs and enable them to run alongside your existing rules, to ensure continued coverage from rules currently using `V1` or `V2` jobs.
+
+* If you only have 8.2 or earlier versions of {beats} and {agent}, you can continue using the same `V1` and `V2` {ml} jobs and the existing prebuilt {ml} rules. 
```

Comment:
So in this instance, if they have existing v1/v2 jobs and using the prebuilt ML rules for those jobs, they're going to need to duplicate these before updating their detection rules otherwise their rules are going to get updated to point to the new v3 jobs and stop working.

I wonder if it'd be useful to have this specific tidbit as an info/warning callout above just so it's explicitly clear 1. what configurations are affected, and 2. what needs to be done before updating their rules to ensure continued coverage. What do you think?

Reply:
@spong Thanks for your detailed comments! I agree, adding a callout above all this might help create more of a call to action, and better establish the conditions involved.

I'm going to take another crack at all of this based on your and @benironside's feedback so far, and I'll ping you when another draft is ready. I think I was leaning too heavily on the previous docs for the transition from `V1` to `V2`, and these require their own approach.